rln-js/index.html

```diff
@@ -294,35 +304,67 @@
         updateFields()
     }
 
-    const ContentTopic = "/toy-chat/2/luzhou/proto";
-    const decoder = new DecoderV0(ContentTopic);
-
     let messages = [];
 
-    const updateMessages = (msgs, div) => {
-        div.innerHTML = "<ul>"
-        messages.forEach(msg => div.innerHTML += "<li>" + msg + "</li>")
-        div.innerHTML += "</ul>"
+    const updateMessages = () => {
+        messagesDiv.innerHTML = "<ul>"
+        messages.forEach(msg => {
+            messagesDiv.innerHTML += `<li>${msg.msg} - [epoch: ${msg.epoch}, proof: ${msg.proofState} ]</li>`
+
+            console.log("msg.verify", msg.verify)
+            console.log("msg.proof", msg.rateLimitProof)
+
+            if (msg.proofState === "verifying...") {
+                console.log("Verifying proof")
+                const res = msg.verify()
+                console.log("proof verified!", res)
+                if (res === undefined) {
+                    msg.proofState = "no proof attached"
+                } else if (res) {
+                    msg.proofState = "verified."
+                } else {
```

Comment:
I am now getting `false` not sure why. I thought at first it was because memberships were not added to the `rlnInstance` but still getting `false` even after adding them.

Help @richard-ramos